app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -216,7 +228,12 @@ class DartdocJobProcessor extends JobProcessor {
         // These package versions are worth investigating, but we don't need
         // alerts on them.
         _logger.warning('Missing dartdoc report when error was present: $job.');
-        await _storeScoreCard(job, _emptyReport());
+        await _storeScoreCard(
+            job,
+            _emptyReport(
+              title: 'Failed to run dartdoc',
+              description: 'The dartdoc process timed out.',
```

Comment:
Do we have a way to indicate the time it took? Should we?

Reply:
This is a tricky one: we have dartdoc runs that neither compete, nor time out (the timeout doesn't kill the process). Once we use the proper kill signal to do it (in this new release), I hope this class of error is gone and we can remove this block.